zb-db/src/test/java/io/camunda/zeebe/db/impl/ColumnFamilyTest.java

```diff
@@ -369,8 +366,8 @@ public void shouldThrowOnMissingForeignKeyInKeyPosition() {
 
     // then
     assertThatThrownBy(() -> columnFamilyWithForeignKey.insert(foreignKey, value))
-        .hasRootCauseInstanceOf(ZeebeDbInconsistentException.class)
-        .hasStackTraceContaining("Foreign key");
+        .isInstanceOf(ZeebeDbInconsistentException.class)
+        .hasMessageContaining("Foreign key");
```

Comment:
:wrench: Not sure it's all that important to match the message if we match the error type directly. I would omit this, I don't see testing the message as particularly useful

Reply:
I'd leave it since `ZeebeDbInconsistentException` is also used for the precondition checks and the only way to distinguish them is via the message.